base_geoengine_jsonify/models/models.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 ACSONE SA/NV.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import geojson
+
+from odoo import models
+from  odoo.addons.base_geoengine.fields import GeoField
+
+
+class Base(models.AbstractModel):
+    _inherit = "base"
+
+    def _jsonify_value(self, field, value):
+        """Adds support for geo_point fields."""
```

Comment:
Let's be a bit more generic, geojson.dumps should support multiline string and polygones too.

```suggestion
        """Adds support for geometry fields."""
```